css-layout-classes/css-layout-classes/styles.css

```diff
@@ -0,0 +1,24 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-onethird {
+  flex-basis: calc(100% / 3);
+}
+
+.column-onefourth {
+  flex-basis: 25%;
+}
+
+.column-twothirds {
+  flex-basis: 66%;
```

Comment:
This one is a bit tricky, but you could do something like this to get the exact 2/3 width `flex-basis: calc(200% / 3)` or `calc(100% * 2 / 3)`